### PR TITLE
10060 story

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: shared/src/business/useCases/trialSessions/serveThirtyDayNoticeInteractor.ts [hunk 1]
```diff
@@ -17,30 +18,37 @@ import { getCaseCaptionMeta } from '../../utilities/getCaseCaptionMeta';
...
   }
 
   const trialSession = await applicationContext
     .getPersistenceGateway()
     .getTrialSessionById({
       applicationContext,
-      trialSessionId: request.trialSessionId,
+      trialSessionId,
     });
+  if (!trialSession) {
```
Comment (ujahio) [inline shared/src/business/useCases/trialSessions/serveThirtyDayNoticeInteractor.ts]: Can we possibly remove one of the exceptions thrown?

File: web-client/integration-tests/petitionsClerkUpdatesATrialSessionAndVerfiesPaperPdfLinks.test.ts [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+      shouldShowIrsNoticeDate: false,
+    });
+
+    petitionsClerkServesPaperCaseToIRS(cerebralTest);
+
+    it('track docket number', () => {
+      cerebralTest.casesReadyForTrial = [cerebralTest.docketNumber];
+    });
+
+    // loginAs(cerebralTest, 'docketclerk@example.com');
```
Comment (ujahio) [inline web-client/integration-tests/petitionsClerkUpdatesATrialSessionAndVerfiesPaperPdfLinks.test.ts]: ❓ :  Are the comments still needed?

File: web-client/src/presenter/actions/Navigation/navigateToPathActionFactory.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export const navigateToPathActionFactory =
```
Comment (ujahio) [inline web-client/src/presenter/actions/Navigation/navigateToPathActionFactory.ts]: ❓: Is this file/factory still useful? Just curious

File: web-client/integration-tests/petitionsClerkUpdatesATrialSessionAndVerfiesPaperPdfLinks.test.ts (hunk 2 above)
Comment (codyseibert) [reply web-client/integration-tests/petitionsClerkUpdatesATrialSessionAndVerfiesPaperPdfLinks.test.ts]: nope, I'll remove them

File: web-client/src/presenter/actions/Navigation/navigateToPathActionFactory.ts (hunk 3 above)
Comment (codyseibert) [reply web-client/src/presenter/actions/Navigation/navigateToPathActionFactory.ts]: nope, will remove

**Review by pixiwyn — DISMISSED** [review]
Test approval

**Review by zachrog — APPROVED** [review]

**Review by jimlerza — APPROVED** [review]

**Pull request merged**